chains/orchestrator-paras/runtime/flashbox/src/lib.rs

```diff
@@ -199,7 +199,21 @@ pub mod currency {
 ///   - Setting it to `0` will essentially disable the weight fee.
 ///   - Setting it to `1` will cause the literal `#[weight = x]` values to be charged.
 pub struct WeightToFee;
-impl WeightToFeePolynomial for WeightToFee {
+impl frame_support::weights::WeightToFee for WeightToFee {
+    type Balance = Balance;
+
+    fn weight_to_fee(weight: &Weight) -> Self::Balance {
+        let time_poly: FeePolynomial<Balance> = RefTimeToFee::polynomial().into();
+        let proof_poly: FeePolynomial<Balance> = ProofSizeToFee::polynomial().into();
+
+        // Take the maximum instead of the sum to charge by the more scarce resource.
+        time_poly
+            .eval(weight.ref_time())
+            .max(proof_poly.eval(weight.proof_size()))
```

Comment:
Flashbox tests are failing because the new fee is lower, how is that possible if we take the max here?

```
 FAIL   dev_flashbox  suites/common-tanssi/fees/test_fee_burning.ts > 🗃️  COMMO1001 Fee  burning test suite > 📁  COMMO1001E01 0% of Fees are burned, they are sent to treasury
AssertionError: expected 165480145n to equal 328960145n

- Expected
+ Received

- 328960145n
+ 165480145n
```

Reply:
So the test is wrong? That sounds bad, that test is supposed to replicate be the same behavior that RPCs use to calculate the fee. If it fails, it can mean we broke fee estimation...